lldb/source/Plugins/TypeSystem/Swift/TypeSystemSwiftTypeRef.cpp

```diff
@@ -1575,22 +1575,47 @@ swift::Demangle::NodePointer TypeSystemSwiftTypeRef::GetDemangleTreeForPrinting(
   return GetNodeForPrintingImpl(dem, node, flavor, resolve_objc_module);
 }
 
-/// Determine wether this demangle tree contains a node of kind \c kind.
+/// Determine wether this demangle tree contains a node of kind \c kind and with
+/// text \c text (if provided).
 static bool Contains(swift::Demangle::NodePointer node,
```

Comment:
Side note: this looks dangerous to me, since any type could appear in a parameter of a function type...

Reply:
Sorry I don't understand the comment. How can this break with a function type?